test/components/HabitList.spec.js

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import TestUtils from 'react-addons-test-utils';
+import {
+  describeWithDOM,
+  mount,
+  spyLifecycle
+} from 'enzyme';
+
+import { HabitList } from '../../client/components/HabitList';
+import { HabitCard } from '../../client/components/HabitTabs';
+
+describe('(Component) HabitTabs', () => {
+  describe('with shallow rendering...', () => {
+    const props = {
+      onMount: sinon.spy(),
+      filteredSchedules: [{
+        "id":5,"habit":{
+          "id":5,"title":"Habit 5","description":"Habit Description 5","created_at":"2016-01-18T20:53:45.932Z","updated_at":"2016-01-18T20:53:45.932Z","goal_id":5,"active":true
+        },
+        "date":"2016-01-18T20:53:45.829Z","status":"do","frequency":"day","repeat":true,"created_at":"2016-01-18T20:53:45Z","updated_at":"2016-01-18T20:53:45Z"
+      }],
+      moveHabit: () => {},
+      handleDelete: () => {},
+      handleEdit: () => {}
+    }
+
+    const wrapper = shallow(<HabitList {...props} />);
```

Comment:
'const' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).<br>Expected an identifier and instead saw '<'.<br>Expected ')' and instead saw 'HabitList'.<br>Missing semicolon.<br>'spread/rest operator' is only available in ES6 (use esnext option).<br>Expected an assignment or function call and instead saw an expression.<br>Unclosed regular expression.<br>Unrecoverable syntax error. (33% scanned).